src/pkg/projects/Microsoft.Net.UWPCoreRuntimeSdk/Microsoft.Net.UWPCoreRuntimeSdk.pkgproj

```diff
@@ -15,13 +21,25 @@
 
   <!-- Identity / Reference package content -->
   <ItemGroup Condition="'$(PackageTargetRuntime)' == ''">
-    <File Include="files\build\*">
+    <File Include="contents\identity\Microsoft.Net.UWPCoreRuntimeSdk.props">
       <TargetPath>build</TargetPath>
     </File>
-    <File Include="files\tools\CoreRuntime\*">
+    <File Include="contents\identity\Microsoft.Net.CoreRuntime.targets">
       <TargetPath>tools/CoreRuntime</TargetPath>
     </File>
   </ItemGroup>
 
+  <!-- Reference runtime packages directly so that MSBuild .props / .targets get imported correctly in VS" -->
+  <Target Name="InjectNuGetDependencyOnRuntimePackages"
+          BeforeTargets="GenerateNuSpec"
+          DependsOnTargets="DetermineRuntimeDependencies"
+          Condition="'$(PackageTargetRuntime)' == ''">
+    <ItemGroup>
+      <Dependency Include="@(RuntimeDependency)">
+        <TargetFramework>uap10.0.15138</TargetFramework>
```

Comment:
Don't use this directly, if you rebase your changes with master, you'll see that there is a property with this value now. You should use `$(UAPvNextTFM)` instead.